universe/System.cpp

```diff
@@ -195,13 +195,13 @@ const std::string System::ApparentName(int empire_id, const Universe& u,
 
         if (m_star == StarType::INVALID_STAR_TYPE) {
             std::stringstream s;
-            s << UserString("UNEXPLORED_REGION") << " #" << std::fixed << std::setprecision(0) << ID();
+            s << UserString("UNEXPLORED_REGION") << " #" << std::to_string(ID());
```

Comment:
use `std::string` instead of `std::stringstream`, call `.reserve(...)` with a suitable size first, and then use `+=` or `.append(...)` to avoid extra allocations.

Reply:
acknowledged. will do.

Reply:
https://quick-bench.com/q/-h8v4haY3lbY-EGQMNExY-2ohLU